.github/workflows/cd.yml

```
@@ -94,10 +94,12 @@ jobs:
 
       - name: Create .env file
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:2:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)